besu/src/main/java/org/hyperledger/besu/cli/options/stable/DataStorageOptions.java

```diff
@@ -86,6 +86,8 @@ public static class Unstable {
     @CommandLine.Option(
         hidden = true,
         names = {BONSAI_LIMIT_TRIE_LOGS_ENABLED, "--Xbonsai-trie-log-pruning-enabled"},
+        defaultValue = "true",
```

Comment:
Should this be enabled by default for all profiles? In the The enterprise-private profile probably makes more sense to not prune I think. @non-fungible-nelson What do you think?

Reply:
Probably not. I thought it might be alright initially since it's scoped to BONSAI, but I know we have a follow-on story related to profiles somewhere...I will check if we need to play that at the same time.

Reply:
This is relevant https://github.com/hyperledger/besu/issues/6560

I think this may have been the PR I was thinking about, which doesn't actually touch the profiles directly but had some impact on them https://github.com/hyperledger/besu/pull/6561

Reply:
I disabled this in enterprise-profile 👍  - was surprised to find enterprise was defaulting to BONSAI, addressed that here: https://github.com/hyperledger/besu/pull/7186